x-pack/legacy/plugins/reporting/index.ts

```diff
@@ -88,8 +80,32 @@ export const reporting = (kibana: any) => {
         return fieldFormats.fieldFormatServiceFactory(uiSettings);
       };
 
+      const {
+        __legacy: { config$ },
+      } = server.newPlatform.setup.plugins.reporting as PluginsSetup;
+      const reportingConfig = await config$.pipe(take(1)).toPromise();
+      const config = server.config(); // FIXME for getting legacy configs
+      const kbnConfig = {
+        path: {
+          data: config.get('path.data'),
+        },
+        server: {
+          basePath: coreSetup.http.basePath.serverBasePath,
+          host: config.get('server.host'),
```

Comment:
These properties need to be retrieved from NP services. I spoke to @joshdover and we're not sure if all these values are exposed in the NP currently.

Reply:
https://github.com/elastic/kibana/issues/56231